regression/esbmc-cpp/cpp/ dangling_ptr/test.desc

```diff
@@ -0,0 +1,4 @@
+CORE
+main.cpp
+
+^VERIFICATION SUCCESSFUL$
```

Comment:
There is a memory leak in this program.

Can you please create another test case that enables --memory-leak-leak?